package.json

```diff
@@ -100,6 +130,26 @@
         "title": "Build IoT Edge Solution",
         "category": "Azure IoT Edge"
       },
+      {
+        "command": "azure-iot-edge.buildAndPushSolution",
+        "title": "Build and Push IoT Edge Solution",
+        "category": "Azure IoT Edge"
+      },
+      {
+        "command": "azure-iot-edge.buildAndRunSolution",
+        "title": "Build and Run IoT Edge Solution",
+        "category": "Azure IoT Edge"
+      },
+      {
+        "command": "azure-iot-edge.runSolution",
+        "title": "Run IoT Edge Solution",
+        "category": "Azure IoT Edge"
+      },
+      {
+        "command": "azure-iot-edge.stopSolution",
+        "title": "Stop IoT Edge Solution",
```

Comment:
just come into mind, Actually it will stop edge runtime both for single module scenario and solution mode right?
 So how about rename the title: Stop IoT Edge Simulator?